lib/active_model/serializer/associations.rb

```diff
@@ -29,7 +28,8 @@ class << base
 
       module ClassMethods
         def inherited(base)
-          base._reflections = self._reflections.try(:dup) || []
```

Comment:
so any class that inherits `Associations` must have a `_reflections` accessor? 
can you document that, please? 
I know it's outside the scope of a refactor, but since we have next to none of this type of documentation, I think it will be good to do it as we go.